### ISSUE TITLE
Loss of discussion board post when forgetting a topic title

### ISSUE THREAD

**Issue by bjr70** [MESH-Research/commons#265]
From Alice Lynn McMichael: 

> When posting to the Discussion board, if you forget to add a title to a new Topic before posting, it deletes the entire message and leaves an error message. (I just lost a long class announcement four times in a row because I was distracted). It's a careless mistake on my part, for sure, but it is one that has happened regularly since I started using groups as a LMS for classes over the last several years.
> 
> For future Commons development updates, I would love to request that the Discussion board post a warning message before deleting work at this step.

Can we add this feature?

**Comment by mikethicke** [comment]
Yep, that is annoying. I'd have to investigate the best way of doing this. Probably disabling submission unless there is a topic set.
